core/src/main/resources/openapi/eth2signer.yaml

```diff
@@ -0,0 +1,92 @@
+openapi: 3.0.0
+info:
+  title: 'Eth2Signer'
+  description: 'Sign Eth2 Artifacts'
+  version: '1.0.0'
```

Comment:
Think it would be simpler if this matched the application version. i.e. 0.0.1-SNAPSHOT. Looks like another piece we have to keep in sync though.

Reply:
hmm ... yes, otherwise we can do some token replacement but this sits in source code.

Reply:
Implemented token replacement to keep the project version sync with yaml version.